src/iblt.h

```diff
@@ -122,6 +126,7 @@ class CIblt
 private:
     void _insert(int plusOrMinus, uint64_t k, const std::vector<uint8_t> &v);
 
+    uint64_t version;
```

Comment:
does this really need to be a uint64_t?  I can't imagine we'll ever have more than a handful of versions ...

Reply:
Good question. I thought that maybe we want to use it as a bit field in the future. I de/serialize it in compact size format, which should be just a byte in any case.

Reply:
ok sounds good.